src/modules/scraper/service.ts

```diff
@@ -1,4 +1,4 @@
-import { Injectable, Logger } from "@nestjs/common";
+import { CACHE_MANAGER, Inject, Injectable, Logger } from "@nestjs/common";
```

Comment:
nit: lint warns that these are not used. can we remove them?

Reply:
Yep. Removed 👍